scripts/house_websites.py

```diff
@@ -37,7 +37,7 @@ def run():
         states.append(state)
 
   destination = "legislators/house.html"
-  url = "http://house.gov/representatives/"
+  url = "http://www.house.gov/representatives/"
```

Comment:
I guess the House moved to a `www` canonical domain for its main site? I fixed this to eliminate the need to follow a redirect.